src/plugins/index_pattern_field_editor/public/open_editor.tsx

```diff
@@ -53,11 +55,32 @@ export const getFieldEditorOpener = (coreStart: CoreStart) => (
     const onSaveField = (updatedField: IndexPatternField) => {
       closeEditor();
 
+      const message = i18n.translate('indexPatternFieldEditor.deleteField.savedHeader', {
```

Comment:
As I mentioned above, I think the success toast should be in the flyout container and keep the logic here to a minimum.